pkg/strategy/support/strategy.go

```diff
@@ -439,7 +393,41 @@ func (s *Strategy) Run(ctx context.Context, orderExecutor bbgo.OrderExecutor, se
 	s.orderExecutor = orderExecutor
 
 	// StrategyController
-	s.status = types.StrategyStatusRunning
+	s.Status = types.StrategyStatusRunning
+
+	s.OnSuspend(func() {
+		// Cancel all order
+		if err := s.activeOrders.GracefulCancel(ctx, session.Exchange); err != nil {
+			errMsg := "Not all {s.Symbol} orders are cancelled! Please check again."
```

Comment:
we can use `{s.Symbol}` in go?

Reply:
oh...
i thought we could
sorry, mislead by previous comments

Reply:
done